webserver/views/api/legacy.py

```diff
@@ -51,6 +55,7 @@ def get_high_level(mbid):
 
 
 @api_legacy_bp.route("/<uuid:mbid>/low-level", methods=["POST"])
+@ratelimit()
```

Comment:
I wonder if we should skip this one for now - this will come from our existing submission apps, which don't have any support for ratelimiting.
New versions of these apps should use the /api/v1 endpoint and support ratelimit responses.
If we decide to leave the limit in place we should be OK, since we wait until we calculate features before submitting it, and computation can take 20-30 seconds. Thoughts?

Reply:
I think skipping for now should be ok, if we still have problems after deploying this and there's a lot of requests to this, then we'll look into ratelimiting this as well.